bin/skel.py

```diff
@@ -133,8 +133,9 @@ def new_problem() -> None:
         custom_output = True
 
     # Read settings from the contest-level yaml file.
+    contest_data = {k: str(v) for k, v in contest.contest_yaml().dict().items()}
```

Comment:
Do we want to cast to `str`, or assert that they are `str`?

Reply:
We also use these keys for copy and substitute later on.

Reply:
We always substituted all of them, so people might expect that all keys from the `contest_yaml` are accessible?

Reply:
Fair enough, I updated the comment to match this intent :slightly_smiling_face: